src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -109,12 +109,12 @@
   </ItemGroup>
   <Target Name="_GenerateMonoAndroidEnums"
       BeforeTargets="CoreCompile"
-      Inputs="..\Mono.Android\obj\$(Configuration)\android-$(AndroidLatestApiLevel)\mcw\api.xml"
+      Inputs="..\Mono.Android\obj\$(Configuration)\android-$(AndroidLatestStableApiLevel)\mcw\api.xml"
       Outputs="@(_MonoAndroidEnum)">
     <MSBuild
         Condition=" !Exists ('%(_MonoAndroidEnum.Identity)') "
         Projects="..\Mono.Android\Mono.Android.csproj"
-        Properties="Configuration=$(Configuration);AndroidApiLevel=$(AndroidLatestApiLevel);AndroidFrameworkVersion=$(AndroidLatestFrameworkVersion)"
+        Properties="Configuration=$(Configuration);AndroidApiLevel=$(AndroidLatestStableApiLevel);AndroidFrameworkVersion=$(AndroidLatestStableFrameworkVersion)"
```

Comment:
There is a [conceptual problem](https://github.com/xamarin/xamarin-android/pull/1380#issuecomment-371968547) with this change: *If* the new API level contains additional properties to `AndroidManifest.xml`-related custom attributes, *and* we want to bind them, then `Xamarin.Android.Build.Tasks.targets` *needs* to build the appropriate API level, which *might not* be "stable."

Thus, a question: is this a scenario we need to care about/wish to support? If it *is*, then I believe staying with the current `$(AndroidLatestApiLevel)` naming is preferable. If it *isn't* something we wish to support, then this change is fine.

Reply:
I noticed that problem too and my thought was "no, there is no way, we have to bear with the fact that we cannot add any new bits for preview API Levels"

If you don't like that, it needs to change e.g. we make sure to build both stable and latest at any time, whenever they differ.

Reply:
(A similar problem would be that an attribute value could be an enum value which is available only in the preview API.)

Reply:
> we make sure to build both stable and latest at any time, whenever they differ.

*This is already the case*: that's why the `_GenerateMonoAndroidEnums` target calls `<MSBuild/>`, to *ensure* that the specified API level has been built. If the stable and latest differ, the latest will be rebuilt. Always.

You are mentioning functionality which already exists.

Reply:
That's good to know. Probably that explains why build didn't fail at that project. Anyways right now it is fine to keep using (or "switch to" ?) stable as its context. If we get stuck in the future, we should rethink about this. The situation might have changed then.